src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/casedocuments/CaseDocumentHandler.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.casedocuments;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.util.StringUtils;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ContestedUploadedDocumentData;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public abstract class CaseDocumentHandler {
```

Comment:
I'd prefer having a bit more meaningful names for these handlers, CaseDocumentHandler tells me only that it handles case documents, but what are these really doing? on the handle methods?